### PR TITLE
Add Tencent provider

### PR DESCRIPTION
**Pull request by nemca**
### Description
Added initial version of Tencent provider with Instance resources.

### CODE REVIEW

**Review by mlabouardy — CHANGES_REQUESTED** [review]
Thanks a million for the contributions, could you please address my comment :)

**Comment by nemca** [comment]
> Thanks a million for the contributions, could you please address my comment :)

Which one comment?

File: providers/tencent/tencent.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+)
+
+func listOfSupportedServices() []providers.FetchDataFunction {
+	return []providers.FetchDataFunction{
+		cvm.Instances,
+	}
+}
+
+func FetchResources(ctx context.Context, client providers.ProviderClient, db *bun.DB) {
+	for _, fetchResources := range listOfSupportedServices() {
```
Comment (mlabouardy) [inline providers/tencent/tencent.go:19]: can you fetch the resources from other regions, see CIVO integration as an example: https://github.com/tailwarden/komiser/blob/master/providers/civo/civo.go#L33

Comment (mlabouardy) [reply providers/tencent/tencent.go:19]: @nemca this one :)

Comment (nemca) [reply providers/tencent/tencent.go:22]: Done.

**Review by mlabouardy — APPROVED** [review]

**Pull request merged**